templates/compose/linkding.yaml

```
@@ -0,0 +1,20 @@
+# documentation: https://linkding.link/
+# slogan: A self-hosted bookmark manager designed to be minimal, fast, and easy to set up.
+# category: RSS
+# tags: rss, feed
+# logo: svgs/linkding.svg
+# port: 9090
+
+services:
+  linkding:
+    image: sissbruecker/linkding:latest
+    volumes:
+      - './data:/etc/linkding/data:rw'
+    environment:
+      - SERVICE_URL_LINKDING_9090
+      - 'LD_SUPERUSER_NAME=${SUPERUSER_NAME}'
+      - 'LD_SUPERUSER_PASSWORD=${SUPERUSER_PASSWORD}'
```

These variables seems like they are required for the service to work so we can mark them as Required. Required variables must be set before deployment and will be highlighted in Coolify's UI with a red border if empty.

```suggestion
      - 'LD_SUPERUSER_NAME=${SUPERUSER_NAME:?}'
      - 'LD_SUPERUSER_PASSWORD=${SUPERUSER_PASSWORD:?}'
```